lib/src/solver/report.dart

```diff
@@ -130,6 +130,39 @@ class SolveReport {
     }
   }
 
+  /// Displays a two-line message, number of outdated packages and an
+  /// instruction to run `pub outdated` if outdated packages are detected.
+  void reportOutdated() {
+    var outdatedPackagesCount = 0;
+
+    final names = _result.packages.map((id) => id.name);
+    for (var name in names) {
+      var newId = _dependencies[name];
```

Comment:
isn't the contents of `_dependencies[name]` the same `id` as exists in `_result.packages`?

Reply:
Yes, thanks.